<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="btnDrawP1">
     <property name="text">
      <string>绘制P1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnDrawP2">
     <property name="text">
      <string>绘制P2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnExec">
     <property name="text">
      <string>求NFP</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnNest">
     <property name="text">
      <string>排版</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="MyCtrlView" name="openGLWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCtrlView</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">view/myctrlview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
